csharp/tools/ZeroC.Ice.Slice.Tools/ZeroC.Ice.Slice.Tools.props

```diff
@@ -12,32 +12,49 @@
 
     <!-- default values -->
     <Choose>
-        <When Condition="Exists('$(MSBuildThisFileDirectory)..\..\cpp') and $(IceOSName) == 'windows'">
+        <When Condition="Exists('$(MSBuildThisFileDirectory)..\..\..\cpp') and $(IceOSName) == 'windows'">
             <!-- Use the Slice compiler from this source build -->
             <PropertyGroup>
-                <IceHome>$(MSBuildThisFileDirectory)..\..\</IceHome>
-                <IceToolsPath>$(MSBuildThisFileDirectory)..\..\cpp\bin\x64\$(Configuration)\</IceToolsPath>
+                <IceToolsPath>$(MSBuildThisFileDirectory)..\..\..\cpp\bin\x64\$(Configuration)\</IceToolsPath>
             </PropertyGroup>
         </When>
-        <When Condition="Exists('$(MSBuildThisFileDirectory)..\..\cpp')">
+        <When Condition="Exists('$(MSBuildThisFileDirectory)..\..\..\cpp')">
             <!-- Use the Slice compiler from this source build -->
             <PropertyGroup>
-                <IceHome>$(MSBuildThisFileDirectory)..\..\</IceHome>
-                <IceToolsPath>$(MSBuildThisFileDirectory)..\..\cpp\bin\</IceToolsPath>
+                <IceToolsPath>$(MSBuildThisFileDirectory)..\..\..\cpp\bin\</IceToolsPath>
             </PropertyGroup>
         </When>
         <Otherwise>
             <PropertyGroup>
                 <!-- Use the Slice compiler from this NuGet package -->
-                <IceHome>$(MSBuildThisFileDirectory)..\</IceHome>
                 <IceToolsPath>$(MSBuildThisFileDirectory)..\tools\$(IceOSName)-$(IceOSArch)</IceToolsPath>
             </PropertyGroup>
         </Otherwise>
     </Choose>
 
+    <!-- Default SliceCompile settings -->
     <ItemDefinitionGroup>
         <SliceCompile>
             <OutputDir>$(MSBuildProjectDirectory)\generated</OutputDir>
         </SliceCompile>
     </ItemDefinitionGroup>
+
+    <Choose>
+        <When Condition="Exists('$(MSBuildThisFileDirectory)..\..\..\slice')">
```

Comment:
This check seems rather dubious to me and could quite easily leave to unexpected behavior if a user happens to have a folder named slice located at `$(MSBuildThisFileDirectory)..\..\..\slice'`.

Reply:
I guess we better do it the other way around `..\slice` is the NuGet package

```
build\ZeroC.Ice.Slice.props
slice
```

Otherwise we assume is the source distribution layout.